test/hotspot/jtreg/compiler/lib/ir_framework/IRNode.java

```diff
@@ -1970,7 +1970,8 @@ public class IRNode {
 
     public static final String SUBTYPE_CHECK = PREFIX + "SUBTYPE_CHECK" + POSTFIX;
     static {
-        macroNodes(SUBTYPE_CHECK, "SubTypeCheck");
+        String regex = START + "SubTypeCheck" + MID + END;
+        macroNodes(SUBTYPE_CHECK, regex);
```

Comment:
Could you also directly call `beforeMatchingNameRegex()` as we do in mainline?

Reply:
I believe you changed it to `macroNodes` here :)

https://github.com/openjdk/valhalla/pull/1296

Reply:
Okay, I guess it makes sense to treat it as macro node here to limit the compile phases. We should probably also change that in mainline but probably not worth to do it separately in mainline.